lib/iris/tests/test_cdm.py

```diff
@@ -1199,7 +1199,8 @@ def test_save_and_merge(self):
             self.assertEqual(len(merged_cubes), 1, "expected a single merged cube")
             merged_cube = merged_cubes[0]
             self.assertEqual(merged_cube.dtype, dtype)
-            self.assertEqual(merged_cube.data.fill_value, fill_value)
+            # Check that the original masked-array fill-value is *ignored*.
+            self.assertArrayAllClose(merged_cube.data.fill_value, -1e30)
```

Comment:
Why not `self..assertAlmostEqual`?

Reply:
array_allclose applies a fixed absolute precision, 6 decimal places by default.
That doesn't work here, you get :
```
Arrays are not almost equal to 6 decimals

(mismatch 100.0%)
 x: array(-1.0000000150474662e+30, dtype=float32)
 y: array(-1e+30)
```

That's why I prefer the "almost equal" approach -- it has better default behaviour, and is also more flexible in supporting a relative tolerance.